scripts/tests/test_zkai_proof_pressure_slope_table_gate.py

```
@@ -0,0 +1,89 @@
+            self.assertIn("width growth is a cost center", markdown)
+            self.assertIn("d128 to d256 h2 seq32 width axis", markdown)
+            self.assertIn("Go: sequence-axis proof-size signal, with timing caveat", markdown)
+            self.assertIn("Caution: width saving weakens and timing is not a speed win", markdown)
+
+    def test_output_paths_reject_aliasing_and_escape(self) -> None:
+        with tempfile.TemporaryDirectory(dir=gate.EVIDENCE_DIR, prefix=".tmp-slope-table-test-") as tmp:
+            same = gate.pathlib.Path(tmp) / "same"
+            with self.assertRaisesRegex(gate.ProofPressureSlopeTableError, "different files"):
+                gate.reject_same_output_paths((same, same, gate.pathlib.Path(tmp) / "other.md"))
+        with tempfile.TemporaryDirectory() as tmp:
+            with self.assertRaisesRegex(gate.ProofPressureSlopeTableError, "inside"):
```

Ruff (0.15.13)

[warning] Use a single `with` statement with multiple contexts instead of nested `with` statements

(SIM117)